catt/controllers.py

```diff
@@ -207,25 +207,22 @@ def volumedown(self, delta):
     def status(self):
         status = self.cast.media_controller.status.__dict__
 
-        if not status["duration"]:
-            echo("State: {player_state}\n".format(**status))
-            return
+        if status["duration"]:
+            dur, cur = int(status["duration"]), int(status["current_time"])
+            duration = time.strftime('%H:%M:%S', time.gmtime(dur))
```

Comment:
You may want:

~~~
human_time = lambda x: time.strftime('%H:%M:%S', time.gmtime(x))
~~~

Or use a normal def, as it's clearer.